### ISSUE TITLE
crash NullPointerException: Tesselating block model ... Block: appliedenergistics2:drive

### ISSUE THREAD

**Issue by rahangar** [AppliedEnergistics/Applied-Energistics-2#3251]
Went to the nether, and immediately decided to return.  While in the portal, I saw a crash.  Never did re-materialize in the overworld.

After that it crashes while loading the world, so I can't even get back in.  I'm playing the direwolf20 1.12 pack v1.1.0 in SP with server and client on the same machine.

Minecraft:  1.12.2
Forge 14.23.0.25231
AE2:  appliedenergistics2-rv5-stable-2

crash log:
https://pastebin.com/mm9j4p3k

**Comment by rahangar** [comment]
I rolled back to an earlier save (before setting up ME controller for the first time) and everything was fine - I was able to travel to different dimensions.    Then, after I setup ME controller and connected it to everything,  I went to the mining world.  When I returned, the same crash happened.  I saved right before going to the mining world, so I can reproduce the issue for you if you like.

The new crash log:
https://pastebin.com/1wuCw1t3
